boards/arm/lpcxpresso55s16/pinmux.c

```diff
@@ -118,6 +118,35 @@ static int lpcxpresso_55s16_pinmux_init(struct device *dev)
 	pinmux_pin_set(port0, 30, port0_pin30_config);
 #endif
 
+#if DT_HAS_NODE(DT_NODELABEL(flexcomm4)) && \
+    DT_NODE_HAS_COMPAT(DT_NODELABEL(flexcomm4), nxp_lpc_i2c)
```

Comment:
Please fix the leading spaces flagged by checkpatch

Reply:
I can do that, but this is the coding style used for these checks in other files within the tree:
```
$ git grep '^    DT_NODE_HAS_COMPAT'|wc -l
11
```